build.gradle.kts

```diff
@@ -17,10 +17,12 @@ plugins {
 
 repositories {
   mavenCentral()
+  maven("https://jitpack.io")
 }
 
 dependencies {
   detektPlugins("io.gitlab.arturbosch.detekt:detekt-formatting:1.23.1")
+  detektPlugins("com.github.TBD54566975:tbd-detekt-rules:0.0.1")
```

Comment:
not sure if this needs to be added here as well as line 69. my guess is probably no but gonna test tmr morning

Reply:
removed